client/src/app/site/pages/meetings/pages/agenda/modules/topics/services/topic-pdf.service/topic-pdf.service.ts

```diff
@@ -171,6 +171,7 @@ export class TopicPdfService {
                     return true;
                 }
             })
+            .filter((singleResult: VotingResult) => !!singleResult.vote)
```

Comment:
You could just do this check within the other filter call. 

Also in line 179 there is logic to handle such cases. So either remove that logic or adjust it. I've committed the latter to you PR as it does not come with behavioral changes opposing to your solution.

Reply:
Yes, it is removed. I have looked at your comment and think, it could work that way.